safebrowser.go

```diff
@@ -492,16 +499,18 @@ func (sb *SafeBrowser) LookupURLs(urls []string) (threats [][]URLThreat, err err
 // updater is a blocking method that periodically updates the local database.
 // This should be run as a separate goroutine and will be automatically stopped
 // when sb.Close is called.
-func (sb *SafeBrowser) updater(period time.Duration) {
-	ticker := time.NewTicker(period)
-	defer ticker.Stop()
-
+func (sb *SafeBrowser) updater(delay time.Duration) {
 	for {
+		sb.log.Printf("Next update in %v", delay)
+		ticker := time.After(delay)
```

Comment:
Instead of declaring a variable, you can just move this into the case statement below:
`case <-time.After(delay)`